sdk/scheduler/src/main/java/com/mesosphere/sdk/http/EndpointsResource.java

```diff
@@ -29,14 +30,13 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * A read-only API for accessing information about how to connect to the service.
  */
 @Path("/v1/endpoints")
 public class EndpointsResource {
-    private static final Logger LOGGER = LoggerFactory.getLogger(EndpointsResource.class);
+    private static final Logger LOGGER = LoggingUtils.getLogger(EndpointsResource.class);
```

Comment:
Another inconsistency / question. What is the difference between using `.class` and `getClass()`?

Reply:
In some classes, we use a private static final logger and use `.class`.

In other classes, we use a per instance logger and use `.getClass()`.

Reply:
Another difference, in the case of an abstract class, is that `getClass()` will result in logging as the final concrete class while `.class` will result in logging as the abstract class.

Reply:
Once again I may have been comparing a subset. Thanks for the clarification.